packages/scaffolding-cli/templates/src/ssr/pages/posts/[slug].tsx

```diff
@@ -0,0 +1,54 @@
+import NextError from "next/error"
```

Comment:
Failing to load posts:

```
commons.1489264ce7b3e3cdf97e.js:1 Uncaught (in promise) Error: The provided `as` value (/web/stacks/posts/documentation-demo) is incompatible with the `href` value (/posts/[slug]). Read more: https://err.sh/zeit/next.js/incompatible-href-as
    at commons.1489264ce7b3e3cdf97e.js:1
    at new Promise (<anonymous>)
    at t.value (commons.1489264ce7b3e3cdf97e.js:1)
    at t.value (commons.1489264ce7b3e3cdf97e.js:1)
    at Object.d.forEach.p.<computed> [as push] (commons.1489264ce7b3e3cdf97e.js:1)
    at s.t.linkClicked (aef9b7b0afdcef12e422e9bf65efc23486568550.92412acc10a488f5be63.js:1)
    at onClick (aef9b7b0afdcef12e422e9bf65efc23486568550.92412acc10a488f5be63.js:1)
    at Object.o (framework.96c24fa20c3269268be5.js:1)
    at p (framework.96c24fa20c3269268be5.js:1)
    at framework.96c24fa20c3269268be5.js:1
```

Reply:
https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%206?workitem=1985

Reply:
Bugs added.